meta-buildpacks/nodejs-function/buildpack.toml

```
@@ -19,11 +19,6 @@ version = "0.7.4"
 id = "heroku/nodejs-npm"
 version = "0.4.4"
```

Since this is a technically a breaking change (eg removal of `CUSTOM_TSCONFIG` support, even if it's not really an issue in practice) it seems this needs a major version bump (since the automation has pre-emptively bumped the versions, but only for a patch release).

In addition to this file `meta-buildpacks/nodejs/buildpack.toml` will also need updating (but the affected lines are out of the range on which GitHub will let me comment).

```suggestion
version = "0.5.0"
```

---

Your suggested change seems to be bumping `heroku/nodejs-npm` version. Did you mean to bump `heroku/nodejs-function` instead? The latter would make sense to me. The meta-buildpacks have a signifcant change warranting a major (or minor, since we're pre 1.0) bump here, but the `heroku/nodejs-npm` buildpack doesn't have any functionality changes.

---

Done in a7db428f099433c850e3a7fd000618c3e0c21deb.

---

Oh sorry I picked the wrong line, yeah what you said :-)